src/mmw/js/src/modeling/views.js

```diff
@@ -494,19 +502,37 @@ var ModelingResultsWindow = Marionette.LayoutView.extend({
     animateIn: function() {
         var self = this;
 
-        this.$el.animate({ height: '55%' }, 400, function() {
+        this.$el.animate({ height: '55%', 'min-height': '300px' }, 200, function() {
             self.trigger('animateIn');
             App.map.set('halfSize', true);
+            $(self.ui.toggle.selector).blur()
+                .find('i')
+                    .removeClass('fa-angle-up')
+                    .addClass('fa-angle-down');
         });
     },
 
     animateOut: function() {
         var self = this;
 
-        this.$el.animate({ height: '0%' }, 100, function() {
+        // Change map to full size first so there isn't empty space when
+        // results window animates out
```

Comment:
`invalidateSize` does get called when we change the map container size in the next line: https://github.com/WikiWatershed/model-my-watershed/pull/309/files#diff-416520c38e4c168e3f58d3f5089772c4R520. Would calling it again after the other animation has ended be of any value?